draft-ietf-jsonpath-base.md

```diff
@@ -282,9 +284,11 @@ Selector:
   consisting of child nodes of the input value.
 
 Singular Query:
-: A JSONPath expression built from segments each of which, regardless of the input value,
-  produces a nodelist containing at most one node.
-  Note: This definition is based on the syntactical definition in {{filter-selector-syntax}}; JSONPath expressions that always produce a singular nodelist but do not conform to that syntax are not Singular Queries.
+: A JSONPath expression built from segments that have been syntactically restricted in
+  a certain way ({{filter-selector-syntax}}) each of which, regardless of the input
```

Comment:
Not sure if this improves, but maybe a "so that" glues the purpose together.
Approved either way.